static/src/javascripts/components/enhancer/enhancer.js

```diff
@@ -1,8 +1,4 @@
-define([
-    'require'
-], function (
-    require
-) {
+define(function () {
```

Comment:
It entirely depends how it is used - it is static when the first argument is a string and there is no callback, and dynamic when the first argument is an array or a string with a callback.